provider/k8s/Gopkg.toml

```diff
@@ -0,0 +1,20 @@
+# The k8s go-discover provider uses Dep because that was the only vendoring
+# tool with the flexibility and stability that worked with the k8s client
+# libraries and allowed us to express only retrieving those libraries.
+#
+# To update:
+#
+#   dep ensure
+#
+
+ignored = [
+  "github.com/hashicorp/go-discover",
+]
+
+[[constraint]]
+  name = "github.com/ericchiang/k8s"
```

Comment:
Since this isn't the official kubernetes client, we will just need to make sure we are able to keep this up to date. (just a note)

Reply:
Yep, true. I still use dep because I realized we need every dependency for the CI to work (TeamCity).